Images/index.html

```diff
@@ -0,0 +1,326 @@
+      <section class="work-container33">
+         <div class="desk_main_img1"></div>
+         <div class="desk_main_cont">
+            <h1>Multi-Post Stories</h1>
+            <div class="my-canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="my-buttons">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <button class="see-button">See Project</button>
+         </div>
+      </section>
+      <section class="work-container44">
+         <div class="desk_main_img1"></div>
+         <div class="desk_main_cont">
+            <h1>Multi-Post Stories</h1>
+            <div class="my-canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="my-buttons">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <button class="see-button">See Project</button>
+         </div>
+      </section>
+      <!-- <section class="ams">
+<h1>About Myself</h1>
+<p>
+Hello I'm a software developer! I can help you build a product, feature or website
+Look through some of my work and experiance! If you like what you see and have a project you need coded, don't hestiate to contact me.
+</p>
+<div class="letconnect_text">LET'S CONNECT</div>
+<div class="letconnect_icons">
+<ul>
+<li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+<li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+<li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+<li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+<li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+</ul>
+</div>
+<button>Get my resume</button>
+<div class="lfs">
+<ul>
+<li><a href="#"><img src="Images\about_languages.svg" alt="" width="327px" height="72px"></a></li>
+<ul>
+<li><a href="#"><img src="Images\about_javascript.svg" alt="" width="303px" height="66.72px"></a></li>
+<li><a href="#"><img src="Images\about_html.svg" alt="" width="303px" height="66.72px"></a></li>
+<li><a href="#"><img src="Images\about_css.svg" alt="" width="303px" height="66.72px"></a></li>
+</ul>
+<li><a href="#"><img src="Images\about_frameworks.svg" alt="" width="327px" height="72px"></a></li>
+<li><a href="#"><img src="Images\about_skills.svg" alt="" width="327px" height="72px"></a></li>
+</ul>
+</div>
+</section> -->
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase as it makes your code  unclean :+1:

Reply:
Its already fixed based on the previous comment.